### PR TITLE
qe: absorb dml crate into prisma-models

### PR DESCRIPTION
**Pull request by tomhoule**
The two crates are no longer distinct in purpose. That simplifies the dependency graph.

### CODE REVIEW

File: query-engine/prisma-models/src/field/scalar.rs [hunk 1]
```diff
@@ -181,3 +183,73 @@ impl From<(InternalDataModelRef, walkers::IndexFieldWalker<'_>)> for ScalarField
         }
     }
 }
+
+pub fn dml_default_kind(default_value: &ast::Expression, scalar_type: Option<ScalarType>) -> DefaultKind {
```
Comment (pimeys) [inline query-engine/prisma-models/src/field/scalar.rs:187]: maybe this could be `pub(crate)`?

**Review by pimeys — APPROVED** [review]
yoinks

**Review by Druue — APPROVED** [review]
lgtm ✨

File: query-engine/prisma-models/src/field/scalar.rs (hunk 1 above)
Comment (tomhoule) [reply query-engine/prisma-models/src/field/scalar.rs:187]: unfortunately not, it's used in `dmmf`.

**Pull request merged**